pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -721,7 +721,7 @@ func (c *VolumeZoneChecker) predicate(pod *v1.Pod, meta PredicateMetadata, nodeI
 
 	nodeConstraints := make(map[string]string)
 	for k, v := range node.ObjectMeta.Labels {
-		if k != v1.LabelZoneFailureDomain && k != v1.LabelZoneRegion {
```

Comment:
@msau42 I recall you saying that the predicate here should only check the beta labels but I can't remember why. May have been because NodeAffinity has the correct values now and label predicate is deprecated? Can confirm if that's still the case?